### ISSUE TITLE
problem about train the Brandwish dataset using faster rcnn

### ISSUE THREAD

**Issue by swearos** [Russell91/TensorBox#94]
I want reproduction the result using faster rcnn which note in your paper,but the result is pool.

1. the AP is about **0.4**,and the **train loss** shocks between 0.2 and 0.6 at the beginning .
2. train about 70000 iterations,but the situation doesn't change.
3. try to change the nms by seting RPN_NMS_THRESH = 0.25 ,but the result seems  change little.

can you give your idea or suggestion to solve the issue?
Thank you!

**Comment by Russell91** [comment]
As I recall we used standard settings with only a small amount of tinkering. If training loss is highly noisy, just try lowering the learning rate.
